app/middleware/auth.js

```diff
@@ -5,7 +5,7 @@ let authenticate = function (req, res, next) {
   if (token) {
     jwt.verify(token, process.env.secret, function (err, decoded) {
```

Comment:
**Nitpick - Code quality**
Your `process.env` is bare. Could you move it to a config file and require it anywhere you need it via `config.secret`. Looks cleaner that way right?